### DISCUSSION TITLE
Release Note Clarifications for version 0.6.2

### DISCUSSION THREAD

**Discussion by dscotese** [ACINQ/eclair#2000] [Node operators]
If anyone feels that all versions of release notes should be included in the same discussion, let me know and I'll remove "for version 0.6.2" from the title.

I'm starting this discussion because the release notes say "When updating eclair, the relay fees for your existing channels will be reset to the value from your eclair.conf."  My conf file has 1000 msat base fee and 100 fee-proportional-millionths, and it looks like the update reset all my channels to 1000 base fee and 200 proportional.  I am using Ride-The-Lightning LN node manager so I used eclair-cli to verify that the proportional rate is twice what it should be, according to the release notes.

**Comment by t-bast** [comment]
> My conf file has 1000 msat base fee and 100 fee-proportional-millionths

Can you share your config? It's likely that you're not setting these correctly.
See https://github.com/ACINQ/eclair/blob/master/eclair-core/src/main/resources/reference.conf
These should be set via `eclair.relay.fees.public-channels.fee-base-msat` and `eclair.relay.fees.public-channels.fee-proportional-millionths`.
Note that there is also a dedicated section for private channels if you want different fees for them.

**Reply by dscotese** [reply]
With some parts redacted (`...`):
```
eclair {
	api {
	    enabled = true // disabled by default for security reasons
	    binding-ip = "0.0.0.0"
	    port = ...
	    password = ...
	  }
	tor.enabled = true
	# tor.auth = safecookie
	tor.auth = password
	tor.password = ...     # Needed if you set auth to password
	socks5.enabled=true
	socks5.port=9050
	socks5.randomize-credentials=true
	chain = mainnet
	bitcoind.rpcport = ...
	bitcoind.rpcuser = ...
	bitcoind.rpcpassword = ...
	zmqblock = "tcp://127.0.0.1:29000"
	zmqtx = "tcp://127.0.0.1:29000"
	node-alias = "Scotese Lightning node"
	node-color = d818f0
	fee-base-msat = 1000
	fee-proportional-millionths = 100
	server.public-ips.0 = 76.174.129.203
	max-to-local-delay-blocks = 2016
        on-chain-fees.feerate-tolerance {
	  ratio-low = 0.1 # will allow remote fee rates as low as half our local feerate
	  ratio-high = 10.0 # will allow remote fee rates as high as 10 times our local feerate
          }
        }
```

**Reply by thomash-acinq** [reply]
The problem is that you're using the old configuration paths. They changed in #1893. Unfortunately I forgot to add a warning at the time. I'm fixing it in #2012.
